include/TrackContentObjectView.h

```diff
@@ -231,6 +232,7 @@ protected slots:
 	bool mouseMovedDistance( QMouseEvent * me, int distance );
 	TimePos draggedTCOPos( QMouseEvent * me );
 	int knifeMarkerPos( QMouseEvent * me );
+	void setColor(const QColor color);
```

Comment:
Top-level const qualifications on parameters have no effect on the function prototype, so are generally omitted for brevity. Or did you mean to pass by const reference instead?

Reply:
Guess I meant const reference, but I've changed to const pointer now